wp-content/themes/core/assets/pcss/typography/anchors.pcss

```diff
@@ -0,0 +1,44 @@
+/* -------------------------------------------------------------------------
+ *
+ * Typography: Anchors
+ *
+ * ------------------------------------------------------------------------- */
+
+a {
+
+	&:focus-visible {
```

Comment:
Still exploring a couple other ways of implementing focus-vislble in a way that meets contrast requirements on any background/element color. Doing that alongside the reset in my next PR